setup.py

```diff
@@ -912,15 +912,6 @@ def run(self):
 
 class my_install_lib(install_lib):
     def install(self):
-        # This is crazy - in setuptools 61.1.0 (and probably some earlier versions), the
-        # install_lib and build comments don't agree on where the .py files to install can
-        # be found, so we end up with a warning logged:
-        # `warning: my_install_lib: 'build\lib.win-amd64-3.7' does not exist -- no Python modules to install`
-        # (because they are actually in `build\lib.win-amd64-cpython-37`!)
-        # It's not an error though, so we end up with .exe installers lacking our lib files!
-        builder = self.get_finalized_command("build")
-        if os.path.isdir(builder.build_platlib) and not os.path.isdir(self.build_dir):
-            self.build_dir = builder.build_platlib
         # We want a failure to find .py files be an error rather than a warning.
         outfiles = super().install()
         if not outfiles:
```

Comment:
I figured the "error instead of warning on no python file" might still be wanted. But if you think that's no longer necessary w/o exe installers (we should add tests to run post_install and test imports after a local install on a CI build anyway), then this entire class can be removed then.

Reply:
killing this seems reasonable